commands/article.js

```diff
@@ -0,0 +1,28 @@
+const config = require('../config.json');
+const Parser = require('rss-parser');
+const parser = new Parser();
+const random = require('random');
+
+
+
+module.exports.execute = async (client, message, args) => {
+	const feed = await parser.parseURL('https://knightsofacademia.org/category/articles/feed');
+	const r = random.int(0 , feed.items.length);
+	let response = `Here is a article i found called "${(feed.items[r].title)}" here is the link to it ${feed.items[r].link}`;
+	if(args[0]){
+		if(args[0].toLowerCase()==='home'){
+			response = "This is home page of the articles https://kingsofacademacia.org/category/articles";
```

Comment:
This is not the correct link to the website. Use `https://knightsofacademia.org/category/articles`

Also, grammar - how about something like this for consistency with other commands?
`KOA Article Archive: https://knightsofacademia.org/category/articles`